tutorials-and-examples/inference-servers/checkpoints/checkpoint_entrypoint.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+export KAGGLE_CONFIG_DIR="/kaggle"
+INFERENCE_SERVER="jetstream-maxtext"
+BUCKET_NAME=""
+MODEL_PATH=""
+
+print_usage() {
+    printf "Usage: $0 [ -b BUCKET_NAME ] [ -i INFERENCE_SERVER ] [ -m MODEL_PATH ]"
+}
+
+print_inference_server_unknown() {
+    printf "Enter a valid inference server [ -i INFERENCE_SERVER ]"
+    printf "Valid options: jetstream-maxtext"
+}
+
+download_kaggle_checkpoint() {
+    BUCKET_NAME=$1
+    MODEL_NAME=$2
+    VARIATION_NAME=$3
+    MODEL_PATH=$4
+
+    mkdir -p /data/${MODEL_NAME}_${VARIATION_NAME}
+    kaggle models instances versions download ${MODEL_PATH} --untar -p /data/${MODEL_NAME}_${VARIATION_NAME}
+    echo -e "\nCompleted extraction to /data/${MODEL_NAME}_${VARIATION_NAME}"
+
+    gcloud storage rsync --recursive --no-clobber /data/${MODEL_NAME}_${VARIATION_NAME} gs://${BUCKET_NAME}/base/${MODEL_NAME}_${VARIATION_NAME}
+    echo -e "\nCompleted copy of data to gs://${BUCKET_NAME}/base/${MODEL_NAME}_${VARIATION_NAME}"
+}
+
+convert_maxtext_checkpoint() {
+    MAXTEXT_VERSION=jetstream-v0.2.0
```

Comment:
When new releases of MaxText are out, is this going to be a manual update? Or can this be an argument passed in?

Reply:
We can change this to be an argument passed in, and also supporting a default value. When new releases of MaxText are out, we can manual update the default value in this script, and also enable users to pass the new version.

Reply:
We can keep this a manual update for new MaxText versions. Thanks for making it as an argument passed in with default value.